cockroachdb/src/jepsen/cockroach.clj

```diff
@@ -313,6 +306,12 @@
                res#))
        res#)))
 
+;; FIXME: CockroachDB features a custom txn retry protocol, documented in
+;; https://www.cockroachlabs.com/docs/build-a-test-app.html#step-4-execute-transactions-from-a-client
+;; However it seems that the JDBC driver and/or clojure sqldb package does
+;; not enable one to reuse the transaction after an exception was thrown. So
+;; we use a (more overweight) traditional retry loop here instead. This
+;; may reduce performance.
```

Comment:
This is a pretty severe loss of performance in some scenarios (due to the loss of priority ratchets). File an issue to define a clojure/jdbc-friendly version of the restart protocol. 

I think at the JDBC level this is supported with `conn.setSavepoint("cockroach_restart")`. It looks like the clojure wrapper has at least some support for savepoints, although it's not clear to me whether you can create a savepoint with a chosen name or not.